proxy-node-shared/src/main/java/uk/gov/ida/notification/validations/DestinationUriValidator.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.ida.notification.validations;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.net.URI;
+
+public class DestinationUriValidator implements ConstraintValidator<ValidDestinationUri, URI> {
```

Comment:
How does this class differ from `DestinationUriStringValidator`?

Reply:
It operates on a URI instead of a String. Apparently we're using both.